packages/light-apps/src/AddAccount/AddAccount.tsx

```diff
@@ -17,28 +17,35 @@ interface MatchParams {
 
 interface Props extends RouteComponentProps<MatchParams> { }
 
+const MENUS = [
+  {
+    label: 'Generate new account',
+    route: 'generate'
+  },
+  {
+    label: 'Import from JSON keyfile',
+    route: 'json'
+  },
+  {
+    label: 'Import from mnemonic phrase',
+    route: 'phrase'
+  }
+];
+
 export function AddAccount (props: Props) {
-  const activeTab = props.location.pathname.split('/')[4];
+  const activeTab = props.location.pathname.split('/')[4]; // FIXME Is there a better way to do that?
```

Comment:
surprisingly and annoyingly it seems to just be the accepted way of doing this. I think we *could* use query strings instead but I think it'd be inapprorpriate.

useRouter hook (not yet merged) looks promising for other reasons but it'd still just pass the same `history`, `match`, and `location` props.

Reply:
we could replace `<Route path='/accounts/:currentAccount/add/' component={AddAccount} />` with `<Route path='/accounts/:currentAccount/add/:activeTab?' component={AddAccount} />` in Content.tsx and then use `props.match.activeTab` in AddAccount.tsx